### PR TITLE
updated docker image to properly cover llvm

### PR DESCRIPTION
**Pull request by mariari**
At some point rebuild, I forgot the last rm, so the current image being uploaded is about 34~ GB too big

This is after comitting the `rm -rf` changes to the image

### CODE REVIEW

File: scripts/llvm-install.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+tar xf "llvm-9.0.0.src.tar.xz"
+
+cd ./llvm-9.0.0.src
+
+mkdir build
+cd build
+
+cmake ../
+
+# mkdir ../../bin/opt
```
Comment (cwgoes) [inline scripts/llvm-install.sh]: Should we delete this line? Is it left in as a comment?

File: scripts/llvm-install.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+cmake --build . --target install
+
+cd ../..
+
+
+rm llvm-9.0.0.src.tar.xz
+rm -rf llvm-9.0.0.src
+# Cleanup
+# rm -rf "${DOWNLOAD_FILE}" "${TARGET}/" llvm-install.sh
```
Comment (cwgoes) [inline scripts/llvm-install.sh]: Should we delete this line? Is it left in as a comment?

File: scripts/llvm-install.sh [hunk 3]
```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+set -eou pipefail
+
+MIRROR="http://releases.llvm.org"
+
+# Determine architecture
+ARCH=$(dpkg --print-architecture)
+
+# Download
+echo "Downloading llvm-9.0.0.src.tar.xz"
```
Comment (cwgoes) [inline scripts/llvm-install.sh]: Can the version easily be a variable?

Comment (mariari) [reply scripts/llvm-install.sh]: done!

**Pull request merged**